tests/live.rs

```diff
@@ -217,6 +217,7 @@ fn check_live_from_filterlists() {
 
 #[cfg(feature = "resource-assembler")]
 #[test]
+#[ignore = "issues/499"]
```

Comment:
also fixed by #503, this test should not be ignored

Reply:
the bad thing that we have 2 branches now.
We fixed this into `master`, but this PR related only to `0.11.x`.
I've manually cherry-picked and pushed https://github.com/brave/adblock-rust/pull/502 and https://github.com/brave/adblock-rust/pull/503 to `0.11.x` to address this.

I believe we need to clarify our branching strategy here.